js/views/PageNav.js

```diff
@@ -4,7 +4,7 @@ import { View } from 'backbone';
 import loadTemplate from '../utils/loadTemplate';
 import app from '../app';
 import $ from 'jquery';
-import SettingsModal from './modals/settings/Settings';
+import SettingsModal from './modals/Settings/Settings';
```

Comment:
I wouldn't think this folder should be capitalized. Maybe we can change it in the file structure? Do you have strong feelings that it should be capitalized?